source/docs/advanced/production-ready/checklist-before-production.md

```diff
@@ -17,4 +17,4 @@ To ensure that you are good to go, you can go through this checklist :
 * [ ] Ensure your server can serve the application in **HTTPS** mode.
   Front-Commerce is aimed at being exposed in HTTPS when in production mode (`NODE_ENV === "production"`). It will redirect users to an HTTPS url if they try to access a page using HTTP. Cookies are also set with secure mode.
   If you want to overcome this limitation (which we don’t recommend!), you may use the [`FRONT_COMMERCE_UNSAFE_INSECURE_MODE` environment variable](/docs/reference/environment-variables.html#Host).
-* [ ] Ensure that the `robots.txt` has been overriden to allow bots. By default, [it forbids crawling to prevent unexpectedly exposing dev applications](https://gitlab.com/front-commerce/front-commerce/-/blob/main/public/robots.txt).
+* [ ] Ensure that the `robots.txt` has been overwritten which instructs bots what they can access. By default, [it forbids crawling to prevent unexpectedly exposing dev applications](https://gitlab.com/front-commerce/front-commerce/-/blob/main/public/robots.txt).
```

Comment:
I think _overriden_ is the right term (it's the fact that a default file will be used unless the same is created in more prioritized place)